mate-base/mate-applets-meta/mate-applets-meta-1.27.1.ebuild

```diff
@@ -14,9 +14,10 @@ HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
 
 LICENSE="metapackage"
 SLOT="0"
-IUSE="appindicator sensors"
+IUSE="appindicator networkmanager sensors"
 
 RDEPEND="=mate-base/mate-applets-${MATE_BRANCH}*
 	appindicator? ( =mate-extra/mate-indicator-applet-${MATE_BRANCH}* )
+	networkmanager? ( gnome-extra/nm-applet )
```

Comment:
nice! this makes sense. It removes the mental load of remembering to install it every time.

Reply:
I read an issue where it was the recommended choice by the devs so made sense to include it.